packages/cli/tsconfig.json

```diff
@@ -3,7 +3,8 @@
     "incremental": true,
     "declaration": true,
     "importHelpers": false,
-    "module": "commonjs",
+    "module": "node16",
+    "esModuleInterop": true,
```

Comment:
Do these settings apply to `ts-node` when we're picking up the `checkly.config.ts` and `*.check.ts` files? Is there any user facing impact?

Reply:
They shouldn't. We don't put the tsconfig into our build folder

Reply:
I think you're right - good point